eos-updater-flatpak-installer/docs/eos-updater-flatpak-installer.8

```diff
@@ -0,0 +1,100 @@
+.\" Manpage for eos-updater-flatpak-installer.
+.\" Documentation is under the same licence as the eos-updater package.
+.TH man 8 "13 Apr 2017" "1.0" "eos\-updater\-flatpak\-installer man page"
+.\"
+.SH NAME
+.IX Header "NAME"
+eos\-updater\-flatpak\-installer — Endless OS Updater - Flatpak Installer
+.\"
+.SH SYNOPSIS
+.IX Header "SYNOPSIS"
+.\"
+\fBeos\-updater\-flatpak\-installer [\-m \-\-mode\fB] [\-p \-\-pull\fB]
+.\"
+.SH DESCRIPTION
+.IX Header "DESCRIPTION"
+.\"
+\fBeos\-updater\-flatpak\-installer\fP installs downloaded flatpaks on Endless OS
+updates upon rebooting into the new update. It is the part of the system that
```

Comment:
s/rebooting into the new update/booting into the new OS deployment/ (terminology is a little more specific).